CONTRIBUTING.rst

```diff
@@ -104,31 +106,30 @@ Set up to run tests
 *******************
 
 To run the tests for the first time, you will need to do some setup (after
-this, you can just work on your given virtualenv each time)::
+this, any ``uv`` commands will automatically work within the right virtual
+environment)::
 
-    python -m venv .venv
-    . .venv/bin/activate
+    uv sync --all-extras
```

Comment:
This shows the simplicity of using `uv` instead of managing virtualenvs by hand. It's also extremely fast compared to the old `pip install` commands (please try this out locally!)